src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,29 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../Types/todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={completed
+        ? 'TodoInfo TodoInfo--completed'
+        : 'TodoInfo'}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally